microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -35,21 +35,22 @@ An overview of the changes involved, as described in [Migrate to Manifest V3](ht
 * Various minor feature improvements.
 
 
-<!-- ---------------------------------------------------------------------- -->
+<!-- ====================================================================== -->
 ## Manifest timeline for the Chromium browser engine
 
-Chromium has revised the timelines for ending support for Manifest V2.  See:
+Chromium has revised the timelines for ending support for Manifest V2.<!-- todo: add "in/for Chrome"? -->  See:
 
 * [Manifest V2 support timeline](https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline)
 * [The phase-out timeline](https://developer.chrome.com/blog/resuming-the-transition-to-mv3#the_phase-out_timeline) in _Resuming the transition to Manifest V3_.
 
 
-<!-- ---------------------------------------------------------------------- -->
+<!-- ====================================================================== -->
 ## Manifest timeline for Microsoft Edge and Partner Center
+<!-- todo: condense this h2 section a lot -->
 
 To help you plan the migration of your extension from Manifest V2 to Manifest V3, consider the following schedule for support for Manifest V2 and V3 in Microsoft Edge and Microsoft Partner Center.
 
-The Microsoft Edge team will independently decide on MV3 migration timelines for Microsoft Edge Add-ons and share an update here.  We continue to analyze the concerns raised by extension developers and explore the optimal path for the Microsoft Edge Add-ons ecosystem.  Meanwhile, to plan the migration of your extension from Manifest V2 to Manifest V3, see [Manifest timeline for the Chromium browser engine](#manifest-timeline-for-the-chromium-browser-engine), above.
+The Microsoft Edge team will independently decide on MV3 migration timelines for Microsoft Edge Add-ons and share an update here.<!-- todo: revise wording in this h2 section? -->  We continue to analyze the concerns raised by extension developers and explore the optimal path for the Microsoft Edge Add-ons ecosystem.<!-- todo: change to "for Microsoft Edge Add-ons."  'ecosystem' is undefined in these articles; if it's a needed term, define it -->  Meanwhile, to plan the migration of your extension from Manifest V2 to Manifest V3, see [Manifest timeline for the Chromium browser engine](#manifest-timeline-for-the-chromium-browser-engine), above.
```

Comment:
"team will ... share an update here"
below, it says instead:
"we will communicate with extension developers when we plan the deprecation of existing MV2-based extensions."

need to bring together & reconcile these two different, similar statements: we will update this page AND communicate [how?] with [all??] extension developers.
@sivMSFT